compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/TilingInterfaceImpl.cpp

```diff
@@ -2065,6 +2086,19 @@ OnlineAttentionOp::getTiledImplementation(OpBuilder &builder,
 
   tiledOperands.emplace_back(scale);
 
+  // Mask
+  Value attnMask = getMask();
+  if (attnMask != nullptr) {
```

Comment:
I believe you can just do `if(attnMask)`

Reply:
We ran into an issue where if attnMask was <<NULL_VALUE>>, if(attnMask) still passed